siriuspy/siriuspy/fofb/csdev.py

```diff
@@ -271,24 +268,14 @@ def get_hlfofb_database(self):
                 'hilim': 8600, 'unit': 'count'},
             
             # filter configuration
-            'FOFBAccFilter-Sel': {
-                'type': 'enum', 'enums': _et.FILTER_OPT,
-                'value': self.FilterOpt.Unit, 'unit': 'Unit_Switching_Custom'},
-            'FOFBAccFilter-Sts': {
-                'type': 'enum', 'enums': _et.FILTER_OPT,
-                'value': self.FilterOpt.Unit, 'unit': 'Unit_Switching_Custom'},
-            'FOFBAccFilter-SP': {
-                'type': 'float', 'value': 20*[0], 'prec': 5, 'count': 20, 
+            'PSConfigMat-SP': {
+                'type': 'float', 'value': _np.zeros((self.nr_chcv, 23)),
+                'prec': 5, 'count': 20,
                 'unit': 'coef'},
```

Comment:
Should the unit be updated?

I think it would be nice to document somewhere the meaning for each column

Reply:
In the siriuspy/siriuspy/clientconfigdb/types/si_fastorbcorr_psconfig.py template header there is an explanation about the elements of each column/row. Do you think we should add something in csdev too?

Reply:
I don't think it's necessary, that one is enough :)

I suggested this before I saw that.